backend/lib/config/gardener.js

```diff
@@ -55,20 +125,43 @@ function getEnvironmentVariable (env, name, type) {
       return value
   }
 }
+function getValueFromEnvironment (env, envVar, type) {
+  const value = env[envVar]
+  return convertValue(value, type)
+}
+
+function getValueFromFile (filePath, type) {
+  try {
+    if (!fs.existsSync(filePath)) {
+      return undefined
+    }
+    const fileContent = fs.readFileSync(filePath, 'utf8')
+    return convertValue(fileContent, type)
+  } catch (error) {
+    /* ignore */
+  }
+}
+
+function getValueFromEnvironmentOrFile (env, envVar, filePath, type) {
+  const value = getValueFromEnvironment(env, envVar, type)
```

Comment:
Do we need this method? I think it is clearer to inline.
```suggestion
  const value = parseConfigValue(env[environmentVariableName], type)
```

Reply:
✅